src/MacroTools/BookSystem/BookManager.cs

```diff
@@ -18,32 +16,9 @@ public static class BookManager
     /// Registers a <see cref="IBook"/> as being visible to all players.
     /// </summary>
     /// <param name="book">The book to register.</param>
-    /// <param name="whichPlayer">If specified, the Book can only be seen by this player.</param>
-    public static void Register(IBook book, player? whichPlayer = null)
+    public static void Register(IBook book)
     {
       Books.Add(book);
-      var launcherButton = new Button("ScriptDialogButton", BlzGetOriginFrame(ORIGIN_FRAME_GAME_UI, 0), 0)
```

Comment:
I'm not sure if we want to take responsibility for the Launcher button away from the Manager. I can easily imagine future Books that are launched without a Launcher button, e.g. the Augment system.